terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,104 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+terraform {
+  backend "s3" {
+  }
+  required_version = "~> 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+
+module "ecs-service" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/ecs-service?ref=1.0.278"
+
+
+  # Environmental configuration
+  environment             = var.environment
+  aws_region              = var.aws_region
+  aws_profile             = var.aws_profile
+  vpc_id                  = data.aws_vpc.vpc.id
+  ecs_cluster_id          = data.aws_ecs_cluster.ecs_cluster.id
+  task_execution_role_arn = data.aws_iam_role.ecs_cluster_iam_role.arn
+
+  # Load balancer configuration
+  lb_listener_arn                 = data.aws_lb_listener.service_lb_listener.arn
+  lb_listener_rule_priority       = local.lb_listener_rule_priority
+  lb_listener_paths               = local.lb_listener_paths
+  multilb_setup                   = true
+  multilb_listeners               = {
+    "priva-api-lb": {
+          listener_arn            = data.aws_lb_listener.secondary_lb_listener.arn,
+          load_balancer_arn       = data.aws_lb.secondary_lb.arn
+      }
+#    "publ-api-lb": {
```

Comment:
This should not be commented out, endpoints need to be accessible via the public LB.